SingularityExecutor/src/main/resources/logrotate.conf.hbs

```diff
@@ -14,15 +14,46 @@ notifempty
 
 {{#if shouldLogRotateLogFile }}
 {{{ logfile }}} {
-  dateformat {{{ rotateDateformat }}}{{#if logfileExtension}}.{{{ logfileExtension}}}{{/if}}
+  dateformat -{{{ rotateDateformat }}}{{#if logfileExtension}}.{{{ logfileExtension}}}{{/if}}
   olddir {{{ rotateDirectory }}}
+{{#if useFileAttributes}}
+  postrotate
+    NOW="$(($(date +%s%N)/1000000))"
+    LOGSTART=`getfattr --only-values -n user.logstart {{{ logfile }}}`
+    if [ "$LOGSTART" != "" ]; then
+      for filename in "{{{ rotateDirectory }}}/{{{ logfileName }}}-$(date +{{{ rotateDateformat }}})"*
+      do
+        setfattr -n user.logend -v "$NOW" {{{ rotateDirectory }}}/$filename
+        setfattr -n user.logstart -v "$LOGSTART" {{{ rotateDirectory }}}/$filename
+      done
+    fi
+    setfattr -n user.logstart -v "$NOW" {{{ logfile }}}
```

Comment:
Since this is saving as the start time, does this mean that a log file that doesn't have the extended attribute hasn't been written to?